### DISCUSSION TITLE
Is it possible to have a a shape as part of two containers simultaneously?

### DISCUSSION THREAD

**Discussion by italovieira** [terrastruct/d2#1976] [Q&A]
My goal is something like 
![image](https://github.com/terrastruct/d2/assets/8337230/6040b807-e9d8-4979-b7af-2fee0d243aff)

Notice that Charles is within _Developers_ and _Group Audit Team_ and David is part of _Operations_ and _Group Audit Team_.

**Comment by alixander** [comment]
dupe: https://github.com/terrastruct/d2/discussions/328

tldr; unfortunately not. The open-source layout engines we use don't have this capability. We may implement it in our own layout engine (TALA), but the syntax on how to do so is still unclear
